tests/python/relay/test_op_level2.py

```diff
@@ -1687,7 +1692,7 @@ def _has_fast_int8_instructions(asm, target):
             dtypes=fast_int8_dtypes,
         )
         # Check that vector int mult and add instructions are generated.
-        assert "vpmulld" in asm and "vpadd" in asm
+        assert "pmulhw" in asm and "paddd" in asm
```

Comment:
I'm not so familiar the specific instructions, does `pmulhw` and `paddd` still be vectorize instructions in this test?

Reply:
It is vector instruction from the middle of 90'th :) It is MMX working on 64 bits